edgelet/edgelet-http/src/modules.rs

```diff
@@ -160,12 +160,16 @@ where
 
 impl std::convert::From<edgelet_core::ModuleRuntimeState> for ModuleStatus {
     fn from(state: edgelet_core::ModuleRuntimeState) -> Self {
-        let start_time = state.started_at().map(chrono::DateTime::to_rfc3339);
+        // IoT Hub requires UTC timestamps to end with 'Z'; otherwise, the timestamps
+        // will incorrectly be interpreted as local time.
+        let start_time = state
```

Comment:
Are these changes still required? I think it's still good to change it anyway for consistency, but edgeAgent was already parsing it as utc, right?

Reply:
As you mentioned, not required but good for consistency.